### ISSUE TITLE
Open built HTML and PDF automatically

### ISSUE THREAD

**Issue by vincerubinetti** [manubot/manubot#101]
It'd be great to automatically open the `.html` output in the user's default browser upon completion of the build. I'm sure this should be fairly easy to accomplish in Python somehow.

Even better would be to detect if it is already open in a tab, and automatically refresh that tab. That way, the user could get an essentially auto-updated output preview by doing a "watch" build, and keeping Chrome/Firefox open on the side to see the updated file.

**Comment by slochower** [comment]
Yes, this would be nice. I fiddled with this awhile back and always ended up opening a new tab even if the build was already open in another tab. I thought I recorded a few of the various things I tried, but apparently not. The bottom of my "`quick-build.sh`" script that I wrote to do this just has:

```bash
DIR=$(dirname "${1}")
cp "$OUTPUT_PATH" "$DIR/tmp.html"
open "$DIR/tmp.html" &

exit 0;
```
